doc/classes/Crypto.xml

```diff
@@ -147,6 +147,32 @@
 				[/codeblocks]
 			</description>
 		</method>
+		<method name="hmac_digest">
+			<return type="PackedByteArray">
+			</return>
+			<argument index="0" name="hash_type" type="int" enum="HashingContext.HashType">
+			</argument>
+			<argument index="1" name="key" type="PackedByteArray">
+			</argument>
+			<argument index="2" name="src" type="PackedByteArray">
+			</argument>
+			<description>
+				Generates an HMAC digest of src using key. The hash_type parameter is the hashing algorithm that
+				is used for the inner and outer hashes.
+			</description>
+		</method>
+		<method name="hmac_equals">
+			<return type="bool">
+			</return>
+			<argument index="0" name="hm1" type="PackedByteArray">
+			</argument>
+			<argument index="1" name="hm2" type="PackedByteArray">
+			</argument>
+			<description>
+				// Compares two HMACS for equality without leaking timing information in order to prevent timing attakcs.
+				// @see: https://paragonie.com/blog/2015/11/preventing-timing-attacks-on-string-comparison-with-double-hmac-strategy
```

Comment:
```suggestion
				Compares two HMACs for equality without leaking timing information to prevent timing attacks. See [url=https://paragonie.com/blog/2015/11/preventing-timing-attacks-on-string-comparison-with-double-hmac-strategy]this blog post[/url] for more information.
```